router/http/src/main/scala/io/buoyant/router/Http.scala

```diff
@@ -37,7 +38,8 @@ object Http extends Router[Request, Response] with FinagleServer[Request, Respon
     val defaultParams: Stack.Params =
       StackRouter.defaultParams +
         FinagleHttp.param.Streaming(true) +
-        ProtocolLibrary("http")
+        ProtocolLibrary("http") +
+        Decompression(false)
```

Comment:
we should just use `FinagleHttp.param.Decompression` like we do for `Streaming`... makes it slightly clearer